internal/services/hdinsight/hdinsight_kafka_cluster_resource_test.go

```diff
@@ -1289,7 +1289,7 @@ func (r HDInsightKafkaClusterResource) restProxy(data acceptance.TestData) strin
 provider "azuread" {}
 
 resource "azuread_group" "test" {
-  name = "acctesthdi-%d"
+  display_name = "acctesthdi-%d"
```

Comment:
We'll need the `security_enabled` prop here :)

```suggestion
  display_name     = "acctesthdi-%d"
  security_enabled = true
```

Reply:
Thanks!